Gems/PhysX/Code/Editor/ComboBoxEditButtonPair.cpp

```diff
@@ -29,6 +29,25 @@ namespace PhysX
 
             layout->addWidget(m_comboBox);
             layout->addWidget(m_editButton);
+
+            connect(m_comboBox, SIGNAL(currentIndexChanged(int)), this, SLOT(onChildComboBoxValueChange(int)));
```

Comment:
Can use the updated syntax here:
```suggestion
            connect(m_comboBox, &QComboBox::currentIndexChanged, this, &ComboBoxEditButtonPair::onChildComboBoxValueChange);
```

Reply:
IIRC it needs the `QOverload` expression to work too